crates/runtime/src/http/v1/responses.rs

```diff
@@ -0,0 +1,259 @@
+                "tool_choice": "auto",
+                "tools": [],
+                "top_p": 1.0,
+                "truncation": "disabled",
+                "usage": {
+                    "input_tokens": 13,
+                    "input_tokens_details": {
+                        "audio_tokens": null,
+                        "cached_tokens": 0
+                    },
+                    "output_tokens": 11,
+                    "output_tokens_details": {
+                        "accepted_prediction_tokens": null,
+                        "audio_tokens": null,
+                        "reasoning_tokens": 0,
+                        "rejected_prediction_tokens": null
+                    },
+                    "total_tokens": 24
+                }
+            })
+        ))),
+        (status = 404, description = "The specified model was not found"),
+        (status = 500, description = "An internal server error occurred while processing the chat completion", content((
+            serde_json::Value = "application/json",
+            example = json!({
+                "error": "An internal server error occurred while processing the chat completion."
+            })
+        )))
+    )
+))]
+pub(crate) async fn post(
+    Extension(llms): Extension<Arc<RwLock<LLMResponsesModelStore>>>,
+    headers: HeaderMap,
+    Json(req): Json<CreateResponse>,
+) -> Response {
+    let span = tracing::span!(
+        target: "task_history",
+        tracing::Level::INFO,
+        "ai_chat",
+        input = %serde_json::to_string(&req).unwrap_or_default()
+    );
+    span.in_scope(|| tracing::info!(target: "task_history", model = %req.model, "labels"));
+
+    override_task_history_with_traceparent(&span.clone(), &headers);
+
+    let span_clone = span.clone();
+    async move {
+        let model_id = req.model.clone();
+        let stream = req.stream.unwrap_or(false);
+
+        let Some(model) = llms.read().await.get(&model_id).cloned() else {
+            return (StatusCode::NOT_FOUND, format!("model '{model_id}' not found")).into_response();
+        };
+
+        if stream {
+            // Streaming response
+            create_response_sse_response(model, req, span_clone).await
+        } else {
+            // Non-streaming response
+            match model.responses_request(req).await {
+                Ok(response) => {
+                    let message = extract_text(&response);
+                    if !message.is_empty() {
+                        tracing::info!(target: "task_history", parent: &span_clone, captured_output = %message);
+                    }
+                    tracing::info!(target: "task_history", parent: &span_clone,  id = %response.id, "labels");
+
+                    Json(response).into_response()
+                }
+                Err(e) => {
+                    tracing::error!(target: "task_history", parent: &span_clone, "{e}");
+                    tracing::error!("Error from v1/responses: {e}");
```

Comment:
We shouldn't allow API calls to pollute the error message log in Spice - that would allow an attacker to spam the logs with invalid requests to potentially hide other activity.